Incorrect RGW Service value for external mode

rgw_service = constants.RGW_SERVICE
            log.info(f"Service {rgw_service} found and will be exposed")
            # custom hostname is provided because default hostname from rgw service
            # is too long and OCP rejects it
            oc = ocp.OCP(
                kind=constants.ROUTE, namespace=config.ENV_DATA["cluster_namespace"]
            )
            route = oc.get(resource_name="noobaa-mgmt")
            router_hostname = route["status"]["ingress"][0]["routerCanonicalHostname"]
            rgw_hostname = f"rgw.{router_hostname}"
>          oc.exec_oc_cmd(f"expose service/{rgw_service} --hostname {rgw_hostname}")

E           ocs_ci.ocs.exceptions.CommandFailed: Error during execution of command: oc -n openshift-storage expose service/rook-ceph-rgw-ocs-storagecluster-cephobjectstore --hostname rgw.apps.pbyregow-vm01.qe.rh-ocs.com.
E           Error is Error from server (NotFound): services "rook-ceph-rgw-ocs-storagecluster-cephobjectstore" not found

RGW_SERVICE differs for external mode env.

--- a/ocs_ci/ocs/constants.py
+++ b/ocs_ci/ocs/constants.py
@@ -912,7 +912,8 @@ RHEL_WORKERS_CONF = os.path.join(CONF_DIR, "ocsci/aws_upi_rhel_workers.yaml")
 NOOBAA_SERVICE_ACCOUNT = "system:serviceaccount:openshift-storage:noobaa"
 
 # Services
-RGW_SERVICE = "rook-ceph-rgw-ocs-storagecluster-cephobjectstore"
+RGW_SERVICE_INTERNAL_MODE = "rook-ceph-rgw-ocs-storagecluster-cephobjectstore"
+RGW_SERVICE_EXTERNAL_MODE = "rook-ceph-rgw-ocs-external-storagecluster-cephobjectstore"
 
 # Miscellaneous
 NOOBAA_OPERATOR_POD_CLI_PATH = "/usr/local/bin/noobaa-operator"

--- a/tests/conftest.py
+++ b/tests/conftest.py
@@ -1680,7 +1680,10 @@ def rgw_endpoint(request):
     oc = ocp.OCP(kind=constants.SERVICE, namespace=config.ENV_DATA["cluster_namespace"])
     rgw_service = oc.get(selector=constants.RGW_APP_LABEL)["items"]
     if rgw_service:
-        rgw_service = constants.RGW_SERVICE
+        if config.DEPLOYMENT["external_mode"]:
+            rgw_service = constants.RGW_SERVICE_EXTERNAL_MODE
+        else:
+            rgw_service = constants.RGW_SERVICE_INTERNAL_MODE
         log.info(f"Service {rgw_service} found and will be exposed")
         # custom hostname is provided because default hostname from rgw service
         # is too long and OCP rejects it

